### PR TITLE
Update Contributing.md

### PR DESCRIPTION
**Pull request by llxia**
Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -10,9 +10,15 @@ aqa-tests is the central project for AQAvit and contains the definition files fo
...
+The AQAvit project maintains the following source code repositories:
 
-* https://github.com/adoptium/aqa-tests
+* https://github.com/adoptium/aqa-tests - the central project for AQAvit
+* https://github.com/adoptium/TKG - a lightweight test harness for running a diverse set of tests or commands
+* https://github.com/adoptium/aqa-systemtest - system verification tests
+* https://github.com/adoptium/aqa-test-tools - various test tools that improve workflow
+* https://github.com/adoptium/STF - system Test Framework
+* https://github.com/adoptium/bumblebench - microbenchmarking test framework
+* https://github.com/adoptium/run-aqa - run aqa GitHub action
```
Comment (smlambert) [inline Contributing.md:21]:
```suggestion
* https://github.com/adoptium/run-aqa - run-aqa GitHub action
```

File: Contributing.md [hunk 2]
```diff
@@ -30,6 +36,60 @@ Contributor Agreement (ECA) on file.
 For more information, please see the Eclipse Committer Handbook:
 https://www.eclipse.org/projects/handbook/#resources-commit
```
Comment (smlambert) [inline Contributing.md:37]: For ECA, we could additionally reference this guide, which gives more details and is more helpful than the Eclipse resources: https://github.com/adoptium/documentation/blob/main/documentation-documentation/eca-sign-off.adoc

File: Contributing.md [hunk 3]
```diff
@@ -30,6 +36,60 @@ Contributor Agreement (ECA) on file.
...
+- Please sign off on your commit in the footer. This can be automatically added to your commit by passing `-s` to `git commit` (see above example).
+
+5. Push the change into your Git repo:
+```
+git push origin my_new_branch
+```
+
+6. We would like to encourage you to open a pull request early and use `Create draft pull request` option. This allows others to check the PR, get early feedback, and helps create a better end product.
+
+7. Run Grinder to validate your PR. Your can find recordings about how to use Grinder in AQA Lightning Talk Series: https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series.
```
Comment (smlambert) [inline Contributing.md]:
```suggestion
7. If you have been given access to run test jobs in our Jenkins server, run Grinder to validate your PR. Your can find recordings about how to use Grinder in AQA Lightning Talk Series: https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series.  If you do not have access, the reviewers of your PR will run some tests.  Reviewers may ask you to run extra tests depending on what changes you have made in your PR.
```

File: Contributing.md [hunk 4]
```diff
@@ -30,6 +36,60 @@ Contributor Agreement (ECA) on file.
...
+5. Push the change into your Git repo:
+```
+git push origin my_new_branch
+```
+
+6. We would like to encourage you to open a pull request early and use `Create draft pull request` option. This allows others to check the PR, get early feedback, and helps create a better end product.
+
+7. Run Grinder to validate your PR. Your can find recordings about how to use Grinder in AQA Lightning Talk Series: https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series. 
+
+8. Ensure the Grinder passes and provide the Grinder link in the PR comment. And ensure your changes passes auto PR builds that will be applied to your pull request.
```
Comment (smlambert) [inline Contributing.md]:
```suggestion
8. Ensure all related Grinder jobs pass and provide the Grinder links in the PR comment. Your changes must also pass the auto PR builds that will be applied to your pull request.
```

File: Contributing.md [hunk 5]
```diff
@@ -30,6 +36,60 @@ Contributor Agreement (ECA) on file.
...
+
+9. Covert PR to `Ready for review` once the PR is ready.
```
Comment (smlambert) [inline Contributing.md:91]:
```suggestion
9. Convert PR to `Ready for review` once the PR is ready.
```

**Review by smlambert — APPROVED** [review]
Thanks @llxia - this will most certainly help contributors!  (I've added some minor suggestions, but approve either way)

File: Contributing.md [hunk 6]
```diff
@@ -30,6 +36,60 @@ Contributor Agreement (ECA) on file.
...
+- Please sign off on your commit in the footer. This can be automatically added to your commit by passing `-s` to `git commit` (see above example).
+
+5. Push the change into your Git repo:
+```
+git push origin my_new_branch
+```
+
+6. We would like to encourage you to open a pull request early and use `Create draft pull request` option. This allows others to check the PR, get early feedback, and helps create a better end product.
+
+7. If you have permission, please run [Grinder](https://ci.adoptopenjdk.net/job/Grinder/) to validate your PR. Your can find recordings about how to use Grinder in AQA Lightning Talk Series: https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series. If you do not have permission, you can skip this step. One of the reviwers will validate the PR for you during the review process.
```
Comment (smlambert) [inline Contributing.md:87]:
```suggestion
7. If you have permission, please run [Grinder](https://ci.adoptopenjdk.net/job/Grinder/) to validate your PR. Your can find recordings about how to use Grinder in AQA Lightning Talk Series: https://github.com/eclipse-openj9/openj9/wiki/AQA-Lightning-Talk-Series. If you do not have permission, you can skip this step. One of the reviewers will validate the PR for you during the review process.
```

**Comment by llxia** [comment]
Thanks @smlambert . The PR is updated.

**Review by LongyuZhang — APPROVED** [review]
LGTM

**Pull request merged**